README.md

```diff
@@ -82,7 +82,7 @@ pip install "git+https://github.com/pytorch/functorch.git"
 Go to the directory where you have cloned the torchrl repo and install it
 ```
 cd /path/to/torchrl/
-python setup.py install
+python setup.py develop
```

Comment:
Goes away if you import torchrl with a PYTHONPATH that does not coincide with the repo location right?

Reply:
Yes.